tests/tests.yaml

```diff
@@ -3,6 +3,28 @@ answer_tests:
   local_art_004: # PR 3081, 3101
     - yt/frontends/art/tests/test_outputs.py:test_d9p
 
+#copied from boxlib frontend
+  local_amrex_012:
```

Comment:
does it make sense, for the purposes of this PR, to make sure the answer tests pass with this named `'local_boxlib_012`, and same for below, and then change the name to `local_amrex_012` in a follow-on PR? What do you think @Xarthisius?

Reply:
I think this is the last point we're waiting for on this PR, so let me ping @Xarthisius again.